### ISSUE TITLE
Package `enum` not built for Python 3.6

### ISSUE THREAD

**Issue by benjaoming** [learningequality/kolibri#6266]
### Observed behavior

`kolibri-server` failing to start, see: https://github.com/learningequality/kolibri-server/issues/43

As I gather, this happens if `enum34` is installed into the environment or if for some other reason the wrong `enum` library is active.

I can see that `enum` is built for many platforms but not `cp36/Linux` ?

CC: @lyw07 

### Expected behavior

An `enum` package for the matching system platform is found.

### User-facing consequences

Unable to start

### Errors and logs

```
Traceback (most recent call last):
  File "./kolibri/__init__.py", line 10, in <module>
    from .utils import env
  File "./kolibri/utils/env.py", line 1, in <module>
    import logging
  File "/usr/lib/python3.6/logging/__init__.py", line 26, in <module>
    import sys, os, time, io, traceback, warnings, weakref, collections
  File "/usr/lib/python3.6/traceback.py", line 5, in <module>
    import linecache
  File "/usr/lib/python3.6/linecache.py", line 11, in <module>
    import tokenize
  File "/usr/lib/python3.6/tokenize.py", line 33, in <module>
    import re
  File "/usr/lib/python3.6/re.py", line 142, in <module>
    class RegexFlag(enum.IntFlag):
AttributeError: module 'enum' has no attribute 'IntFlag'
Error in sys.excepthook:
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/apport_python_hook.py", line 53, in apport_excepthook
    if not enabled():
  File "/usr/lib/python3/dist-packages/apport_python_hook.py", line 24, in enabled
    import re
  File "/usr/lib/python3.6/re.py", line 142, in <module>
    class RegexFlag(enum.IntFlag):
AttributeError: module 'enum' has no attribute 'IntFlag'

Original exception was:
Traceback (most recent call last):
  File "./kolibri/__init__.py", line 10, in <module>
    from .utils import env
  File "./kolibri/utils/env.py", line 1, in <module>
    import logging
  File "/usr/lib/python3.6/logging/__init__.py", line 26, in <module>
    import sys, os, time, io, traceback, warnings, weakref, collections
  File "/usr/lib/python3.6/traceback.py", line 5, in <module>
    import linecache
  File "/usr/lib/python3.6/linecache.py", line 11, in <module>
    import tokenize
  File "/usr/lib/python3.6/tokenize.py", line 33, in <module>
    import re
  File "/usr/lib/python3.6/re.py", line 142, in <module>
    class RegexFlag(enum.IntFlag):
AttributeError: module 'enum' has no attribute 'IntFlag'
```

### Context

See https://github.com/learningequality/kolibri-server/issues/43

**Comment by intelliant01** [comment]
> As I gather, this happens if `enum34` is installed into the environment or if for some other reason the wrong `enum` library is active.

Confirmed that I have `enum34` installed.

**Comment by jredrejo** [comment]
Solved with #6269
